tools/unicode-check.py

```diff
@@ -10,7 +10,7 @@
 # The set of Unicode code points greater than 127 that we
 # allow in the source code.
 box_drawing_chars = set(chr(cp) for cp in range(0x2500, 0x2580))
-allowed = (set(['®', 'é', 'ê', 'ó', 'ö', 'ü', 'λ', 'π', 'ω', '∫', '≠', '≥']) |
+allowed = (set(['®', 'é', 'ê', 'ó', 'ö', 'ő', 'ü', 'λ', 'π', 'ω', '∫', '≠', '≥']) |
```

Comment:
looks like CI is failing because this line is a little too long now--may need to break it over another line

Reply:
Thanks, I couldn't figure out what went wrong. Line is broken in new commit.